fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -2397,6 +2401,8 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 		    idempotencyIdSet.param1 = kv.key;
 		    idempotencyIdSet.param2 = kv.value;
 		    auto& tags = pProxyCommitData->tagsForKey(kv.key);
+		    ASSERT(!SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST ||
+		           pProxyCommitData->db->get().logSystemConfig.numLogs() == self->tpcvMap.size());
```

Comment:
Would you please add a comment here explaining why it is fine to skip this code (it may not be obvious for those who don't know much about this feature)? Thanks!